workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -193,6 +201,16 @@ private void EnsureLoaded(Type behaviourType)
 
             componentPresentRequirementsForBehaviours[behaviourType] = componentsRequiredPresent.ToList();
             componentAuthRequirementsForBehaviours[behaviourType] = componentsRequiredWithAuthority.ToList();
+
+            var workerTypeAttribute = behaviourType.GetCustomAttribute<WorkerTypeAttribute>();
+            if (workerTypeAttribute != null)
+            {
+                componentWorkerTypeRequirementsForBehaviours[behaviourType] = workerTypeAttribute.WorkerTypes;
+            }
+            else
+            {
+                componentWorkerTypeRequirementsForBehaviours[behaviourType] = null;
+            }
```

Comment:
This if-else can be replaced with:

`componentWorkerTypeRequirementsForBehaviours[behaviourType] = workerTypeAttribute?.WorkerTypes;`

Reply:
neat!